### PR TITLE
feat: boba level

### PR DESCRIPTION
**Pull request by whgdk**

### CODE REVIEW

File: src/components/Levels/index.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export const LEVELS: { [url: string]: JSX.Element } = {
   typeracer: <TypeRacerLevelWrapper />,
   thegreatone: <MrFrog />,
   maze: <JLevel />,
```
Comment (DonLe27) [inline src/components/Levels/index.tsx:18]: Can you go into JLevel and point the nextUrl to your level?

File: src/components/Levels/BobaShop.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const [ isCompleted, setIsCompleted ] = useState(false);
+  const [password, setPassword] = useState('');
+
+  const complete = () => {
+    setIsCompleted(true);
+  };
+
+  const handleSubmit = (evt: { preventDefault: () => void }) => {
+    evt.preventDefault();
+    if (password == 'decaf') {
```
Comment (BryanPan342) [inline src/components/Levels/BobaShop.tsx]: do we want to put the password in firebase?

**Review by BryanPan342 — COMMENTED** [review]
@whgdk  thoughts?

File: src/components/Levels/BobaShop.tsx (hunk 2 above)
Comment (DonLe27) [reply src/components/Levels/BobaShop.tsx]: Ah dang spoiler alert, i should have solved the puzzle first before reading the comments :((

**Review by BryanPan342 — APPROVED** [review]
LGTM THANKS @whgdk and @DonLe27

**Pull request merged**